src/Kafka/MetaDataFromKafka.php

```diff
@@ -108,7 +108,7 @@ public function setClient(\Kafka\Client $client)
      */
     public function listBrokers()
```

Comment:
This seemed like a bug to me.  In this class $this->brokers is initialized to array() not null, and I can't find any place it gets set to null.  I wasn't sure so I added the empty array check, but really I believe the null check should be removed.  Can anyone verify this?